projects/admin/src/app/menu/menu-user/menu-user.component.ts

```diff
@@ -44,7 +42,7 @@ export class MenuUserComponent implements OnInit {
   eventMenuClick(event: MenuItem) {
     // If the user logout, we delete the local storage
     if (event.getAttribute('id') === 'logout-menu') {
-      this._librarySwitchMenuStorageService.removeStorage();
+      this._menuService.logout$.next(true);
```

Comment:
I would not call the next directly but I would use a method in the menuService such as: `this._menuService.logout(true)`